tests/components/freedompro/test_light.py

```diff
@@ -0,0 +1,156 @@
+"""Tests for the Freedompro light."""
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_HS_COLOR,
+    DOMAIN as LIGHT_DOMAIN,
+    SERVICE_TURN_ON,
+)
+from homeassistant.const import ATTR_ENTITY_ID, SERVICE_TURN_OFF, STATE_OFF, STATE_ON
+
+from tests.components.freedompro import init_integration
+
+
+async def test_light_get_state(hass):
+    """Test states of the light."""
+    await init_integration(hass)
+    registry = await hass.helpers.entity_registry.async_get_registry()
```

Comment:
Add:

```py
from homeassistant.helpers import entity_registry as er
```

and fetch the registry directly (without the helper)

```py
registry = er.async_get(hass)
```

There are a couple of more occurrences of this.